### ISSUE TITLE
[BUG] 1.25.0 browsers in the 1.25 Docker image

### ISSUE THREAD

**Issue by eatonjl** [microsoft/playwright#16726]
**Context:**
- Playwright Version: 1.25.0
- Operating System: Linux
- Node.js version: 16.16.0
- Browser: All
- Extra: 

**Describe the bug**

More of a heads up because I am not sure if others have tried to update to 1.25.0 and had a similar experience. I updated Playwright to 1.25.0 and pushed the code to my repository. I then updated my Docker image to be mcr.microsoft.com/playwright:v1.25.0-focal. When I ran the tests it said I needed to install the correct browsers. I am not needing this looked into for my code. I have reverted to 1.24.1 until I can feel confident in 1.25.0. I just thought I would point it out in case anyone else was considering upgrading to 1.25. I am fine closing this once someone from the Playwright team has had eyes on it so they are aware there could be a potential issue.

**Comment by mxschmitt** [comment]
This is caused because your Playwright version does not match with the Docker image version. Make sure that you only have `@playwright/test` and no other Playwright packages inside your package.json. Can you share a repro with us?

**Comment by eatonjl** [comment]
@mxschmitt I cannot right now. I do not have time. It is possible there were other packages, but this was not the first time I have updated and needed to update the Docker image. I made sure to follow the instructions for updating, using `npm install -D @playwright/test@latest`. As I said, I already downgraded to 1.24.1 and will be staying there for the time being. Now that this has had eyes on it, I will go ahead and close it. Its intentions was more of an FYI. Thank you for looking at it!
